generator.go

```diff
@@ -553,6 +553,23 @@ func processType(s *metaSchema, pName, pDesc, path, parentPath string) (typeRef
 				sf.TypePrefix = ""
 				sf.TypeRef = gotType
 				sf.PtrForOmit = true
+			} else if !hasAddlProps {
+				if patternProps := propSchema.PatternProperties; patternProps != nil {
```

Comment:
maybe we can put this condition as a `&&` of the if just above ? wdyt ?

Reply:
sure.